solutions/python-igraph/clique_percolation/clique_percolation.py

```diff
@@ -0,0 +1,64 @@
+import igraph as ig
+import itertools
+import matplotlib.pyplot as plt
+
+EXAMPLE_K = 3
+
+def k_communities(g, k, min_common_vertices):
```

Comment:
I would really call this `clique_percolation` to be consistent with the usual name.